includes/template-library/manager.php

```diff
@@ -63,6 +70,10 @@ public function __construct() {
 		$this->add_actions();
 	}
 
+	public function set_wordpress_adapter( Wordpress_Adapter_Interface $wordpress_adapter ) {
```

Comment:
can we do this in `__construct()` ?

Reply:
Adding this to construct would not create the right instance.

Reply:
weird, but ok

Reply:
@mykytamurzin It could be added in construct. But this was the least intrusive change. A lot of places access the class like this `Plugin::$instance->templates_manager` all over the code base. Can you please sit with @mugurelLupu to just make sure it is safe. It should probably then be added already in the `init_components()` in `plugin.php`